core/options.js

```diff
@@ -250,11 +259,23 @@ Blockly.Options.prototype.getMetrics;
 Blockly.Options.parseMoveOptions_ = function(options, hasCategories) {
   var move = options['move'] || {};
   var moveOptions = {};
-  if (move['scrollbars'] === undefined && options['scrollbars'] === undefined) {
+  if (move['scrollbars'] === undefined) {
```

Comment:
Just confirming that you're intentionally removing support for this: 

```
const options = {
   scrollbars: true
}
```

Reply:
Wasn't my intensions, must have just gotten lost in the refactor as I was working on it. I will add it back.